src/Components/v2/CollectionsHubsNavItem.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Image, Serif } from "@artsy/palette"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+
+interface CollectionsHubsNavItemProps extends SpaceProps {
+  href: string
+  imageUrl: string
+}
+
+export const CollectionsHubsNavItem: React.FC<CollectionsHubsNavItemProps> = ({
+  href,
+  imageUrl,
+  children,
+}) => {
+  return (
+    <OuterLink to={href}>
+      <ImageContainer>
+        <ImageOverlay>
+          <HubImage src={imageUrl} width="100%" alt={"todo fill in"} />
```

Comment:
Could the `alt` tag here just recapitulate the `hub.title`?

Reply:
Yes! Thanks for catching. I'll fix this. In retrospect I'm glad I put in a bad alt so that it would get caught, instead of just leaving it blank.

Reply:
Cool cool, wasn't sure if you were deferring intentionally, since there's probably some complexity here (if we need to account for subtitles, or the title text having inline markup, etc.)

Reply:
I actually think I'm going to fix this as part of the next story (hubs nav on home page), because that one will force me to think about how to pass all the right things into each item.